docs/application/web/guides/security/privacy-related-permissions.md

```diff
@@ -88,6 +88,8 @@ To check whether an application has permission to use a privilege, and to reques
 
     If the decision is definitive, any subsequent `requestPermission()` calls result in an immediate response with an appropriate result: `PPM_ALLOW_FOREVER` or `PPM_DENY_FOREVER`. However, the user can change the status of privacy-related privileges later by modifying the privacy settings on the device. For this reason, the application must always check the status of privacy-related privileges before using protected functionality.
 
+4. From Tizen 5.0 you can check and request multiple privacy privileges at once. To do that please use `checkPermissions` [mobile](../../../../org.tizen.web.apireference/html/device_api/mobile/tizen/ppm.html#PrivacyPrivilegeManager::checkPermissions) and [wearable](../../../../org.tizen.web.apireference/html/device_api/wearable/tizen/ppm.html#PrivacyPrivilegeManager::checkPermissions) and `requestPermissions` [mobile](../../../../org.tizen.web.apireference/html/device_api/mobile/tizen/ppm.html#PrivacyPrivilegeManager::requestPermissions) and [wearable](../../../../org.tizen.web.apireference/html/device_api/wearable/tizen/ppm.html#PrivacyPrivilegeManager::requestPermissions)
```

Comment:
Hello.
Would you please update the link to use the tizen-docs repository?

I think changing `../../../../org.tizen.web.apireference/html/device_api/` into `../../api/latest/device_api` will work.